src/backend/bacon-video-widget-gst-missing-plugins.c

```diff
@@ -521,30 +585,31 @@ show_codec_missing_information_dialog (TotemCodecInstallContext *ctx)
 	                                 GTK_MESSAGE_ERROR,
 	                                 GTK_BUTTONS_CANCEL,
 	                                 _("Unable to play the file"));
+	action = determine_vlc_launch_action_for_missing_codecs (bacon_video_widget_get_mrl (ctx->bvw));
 
-	/* Show additional button pointing to our online store if the
-	 * missing codec can be installed via our codecs activation key. */
-	if (missing_codecs_supported ((const char**)ctx->details)) {
```

Comment:
It is, though I'd prefer to keep that code around until we decide what we want to do with it (since there is quite a lot that could be ripped out ... not sure if we want to try and keep our delta with upstream small).

@cosimoc  thoughts?